### PR TITLE
confluence.cfg.xml little automation

### PR DESCRIPTION
**Pull request by posteingang**
I created a little define to change - or audit the properties in confluence.cfg.xml file.
Unfortunately it is only working when the init setup is completed.
Maybe it helps anyway.

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -4,6 +4,11 @@
 #
 class confluence::install {
 
+  File {
```
Comment (bastelfreak) [inline manifests/install.pp]: This is very dangerous as it might pollute other file resources as well. can you revert this?

**Comment by bastelfreak** [comment]
Hi @posteingang. thanks for the PR. Can you check the inline comment I made and squash the commits down a bit?

File: manifests/install.pp (hunk 1 above)
Comment (posteingang) [reply manifests/install.pp]: I can... but it is also used in the class confluence::config.... so there is no difference?!

Comment (bastelfreak) [reply manifests/install.pp]: yes, but it would be good to not introduce it in more places.

**Comment by bastelfreak** [comment]
Thanks!

**Pull request merged**